Lib/pathlib.py

```diff
@@ -886,6 +886,15 @@ def relative_to(self, *other):
         return self._from_parsed_parts('', root if n == 1 else '',
                                        abs_parts[n:])
 
+    def is_relative_to(self, *other):
```

Comment:
I'd suggest renaming this to `*other_path`

Reply:
ditto